webrtc-stats.html

```diff
@@ -2345,6 +2367,105 @@
             </dl>
           </section>
         </div>
+        <section>
+          <h3>
+            <dfn>RTCNetworkType</dfn> enum
+          </h3>
+          <div>
+            <pre class="idl">
+enum RTCNetworkType {
+    "bluetooth",
+    "cellular",
+    "ethernet",
+    "wifi",
+    "wimax",
+    "other",
+    "unknown"
+};</pre>
+            <table data-link-for="RTCNetworkType" data-dfn-for=
+            "RTCNetworkType" class="simple">
+              <tbody>
+                <tr>
+                  <th colspan="2">
+                    Enumeration description
+                  </th>
+                </tr>
+                <tr>
+                  <td>
+                    <dfn><code>bluetooth</code></dfn>
+                  </td>
+                  <td>
+                    <p>
+                      A Bluetooth connection.
+                    </p>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <dfn><code>cellular</code></dfn>
+                  </td>
+                  <td>
+                    <p>
+                      A cellular connection (e.g., EDGE, HSPA, LTE, etc.).
+                    </p>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <dfn><code>ethernet</code></dfn>
+                  </td>
+                  <td>
+                    <p>
+                      An Ethernet connection.
+                    </p>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <dfn><code>wifi</code></dfn>
+                  </td>
+                  <td>
+                    <p>
+                      A Wi-Fi connection.
+                    </p>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <dfn><code>wimax</code></dfn>
+                  </td>
+                  <td>
+                    <p>
+                      A WiMAX connection.
+                    </p>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <dfn><code>other</code></dfn>
+                  </td>
+                  <td>
+                    <p>
+                      The connection type is known, but isn't one of the
+                      enumerated types.
```

Comment:
What might fall into this category? If something is "other", would it make sense to add it to the spec or is "other" something we want to use?

Reply:
Hopefully nothing with statistical significance. But there certainly are more physical-layer protocols than are listed here.

I'd be up for just merging "other" and "unknown", personally. Different browser versions may have different breakdowns of "other" vs "unknown" which doesn't make it very useful as a standard stat.

Reply:
Merging SGTM

Reply:
Let's merge them - "unknown to stats writer". Might want to add a note that this list is expected to be updated when new connectivity types are defined, and hope that coders who would like to use "other" instead send a PR in our direction.

Reply:
+1 to merge the  unknown/other.